### ISSUE TITLE
Blending bundle needs a bit of cleanup

### ISSUE THREAD

**Issue by chan-hoo** [NOAA-EMC/rrfs-workflow#108]
@delippi Some items I should have caught with your last blending PR, but didn't.  Can you clarify about the removal of BLENDINGPYTHON?  And is shifting those two utility python scripts to USH okay with you?

- The environment `BLENDINGPYTHON` should be removed. (thought was done with most recent blending PR, but see these in current system - Donnie?):

```
exrrfs_blend_ics.sh:     ${BLENDINGPYTHON} exrrfs_blending_fv3.py $Lx $glb $reg $trcr $blend $use_host_enkf
exrrfs_make_ics.sh:  ${BLENDINGPYTHON} exrrfs_chgres_cold2fv3.py $cold $grid $akbk $akbkcold $orog

```

-  The python script `exrrfs_blending_fv3.py` and `exrrfs_chgres_cold2fv3.py` should be moved to `ush` since they are invoked by ex-scripts.  Also, could they be executed directly rather than copied into run directory?  (to be done - Donnie?)
- 
- A new python environment including all necessary packages needs to be set up for RRFS.  

@delippi can you remind me what the BLENDINGPYTHON package includes that the default python installs on WCOSS don't have?

**Comment by delippi** [comment]
Hi @chan-hoo @MatthewPyle-NOAA sorry I must have missed this. 
I think moving the python scripts to the ush directory and executing directly from there is fine. As for removing BLENDINGPYTHON, is there a python environment that would be available that I could try instead? I might also have to check that I can build the fortran codes with it too.

**Comment by MatthewPyle-NOAA** [comment]
Thanks @delippi - on the BLENDINGPYTHON, I'm not sure.  Think I was confused by a commit in your recent PR that made it seem like it had been removed.

**Comment by delippi** [comment]
@MatthewPyle-NOAA , oh I see what I did. When separating blending from make_ics, I didn't know at first how to give the script the BLENDINGPYTHON environment so I had it hardcoded it into the exrrfs_blend_ics.sh. Then I "removed" it from being hardcoded in the script after putting BLENDINGPYTHON in the blend_ics.local.lua file. Sorry, that I made that really confusing.

@chan-hoo, is the BLENDINGPYTHON okay in the *.lua files or does it need removed completely? Do you have any recommendations?

**Comment by delippi** [comment]
Hi @MatthewPyle-NOAA , we have the BLENDINGPYTHON variable taken care of now with PR #254. I tested executing the python scripts directly rather than copied into run directory, but that doesn't work since the *.so files need to be in the same directory as the script. The other suggestion was to move the python blending scripts to the ush directory. I just looked through the NCO standards and I think it would be a good idea. I will go ahead and add that to my PR.
